presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java

```diff
@@ -296,4 +370,29 @@ private long calculateTotalRows(int scaleBase, double scaleFactor)
         }
         return (long) totalRows;
     }
+
+    public static Type getNumericType(TypeSignature numericTypeSignature)
+    {
+        if (numericTypeSignature.getBase().equals(DECIMAL)) {
```

Comment:
I thought there was a discussion about not making the type dynamic for the TPCH connector.

The TPC-H connector should return the expected types according to what the TPC-H spec says. If we want to generate datasets with other types for the purpose of benchmarking or testing, the loading queries should be adjusted to introduce appropriate casts (e.g., in copyTable in the XXXQueryRunners)